CMakeLists.txt

```diff
@@ -30,6 +31,8 @@ set(RPCLIB_VERSION_MAJOR 1)
 set(RPCLIB_VERSION_MINOR 0)
 set(RPCLIB_VERSION_PATCH 0)
 
+add_definitions(-DRPCLIB_DEFAULT_PORT=${RPCLIB_DEFAULT_PORT})
```

Comment:
I'd prefer `target_add_definitions` (you can add it here: https://github.com/rpclib/rpclib/blob/dev/CMakeLists.txt#L135)